streamz/core.py

```diff
@@ -28,6 +28,16 @@
 logger = logging.getLogger(__name__)
 
 
+class ClearMSG:
```

Comment:
You may want a MSG base class so you can just check if an item is an instance if that class